module/meowfficer/collect.py

```diff
@@ -67,38 +68,46 @@ def _meow_detect_shift(self, skip_first_screenshot=True):
                     break
         return flag
 
-    def _meow_is_talented(self):
+    def _get_meow_talent_grid(self) -> Tuple[List[Button], bool]:
         """
-        Examine meowfficer for special talented abilities
-        i.e. no tiers or is rainbow colored
-
+            get meow talent grid list and special talent status
         Returns:
-            bool
+           Tuple[List[ButtonGrid], bool]
         """
         # Wait for complete load before examining talents
         logger.info('Configured to retain this type of meowfficer, '
                     'wait complete load and examine base talents')
+
+        list_talent_grid = []
+        special_skill = False
```

Comment:
`special_skill` vs `special_talent`?

I know minor but curious for the name change from `talented` and also a method has as `special_talent` while another area has it as `special_skill` since both are the same, no?

Reply:
Thanks for the fix! Pet peeve when it comes to minor things like there